### DISCUSSION TITLE
Reg. choosing my first project for preparation for Mentorship Programs in Summer'24

### DISCUSSION THREAD

**Discussion by harshsingh-24** [cncf/mentoring#1092] [Q&A] [answered]
Hi everyone.

I am new to open-source and want to begin my journey with CNCF. I am looking forward for new projects to contribute to so that I can strengthen myself well for the Mentorship Programs of Summer'24. 

I know about [CLOTributer](https://clotributor.dev/search?good_first_issue=true&page=1) which will help me finding `good-first-issue`. But since there are so many projects, I feel overwhelmed by that. I am currently following [Do you want to start contributing to open source and need help figuring out where to begin?](https://contribute.cncf.io/contributors/getting-started/) for finding out a good project to start to. 

Following are my questions - 
1. Can someone please advice me on which projects to choose from - Graduated, Incubating or Sandboxed? 
2. Should I start contributing randomly to every issue of project labeled `good-first-issue`? 
3. How do you know that which projects get selected for mentorship programs and what happens if your project whom you are contributing in doesn't get selected?
4. Is there any other advice which can help me choosing a specific project that will help me in future? 

Thanks.

**Comment by yurishkuro** [comment, marked as answer]
* is the domain where the project operates interesting to you?
* does it have active contributions and community?
* does it have a track record with mentorships? Can you get feedback from past mentees?
* are you familiar with the tech stack of the project to be successful?

In the end, I think *how* you do in the project is more impactful on your career, but if you have certain job prospects already you could try picking the project in a similar areas.

**Reply by harshsingh-24** [reply]
That helps @yurishkuro. Thanks.
